contrib/cirrus/setup_environment.sh

```diff
@@ -65,9 +67,9 @@ case "$SPECIALMODE" in
                 tee -a /etc/environment) && eval "$X" && echo "$X"
             setup_rootless
         fi
+        remove_packaged_podman_files
         ;;
     in_podman)  # Assumed to be Fedora
-        dnf install -y podman
```

Comment:
IIUC the addition of podman to `contrib/cirrus/packer/fedora_setup.sh` is to get conmon. Is that correct? (And if so, could this *please* be documented explicitly in both the fedora_setup.sh file and the remove_packaged_podman_files function?)

If the above is correct, then won't podman-in-podman break now because of missing conmon?

Reply:
Almost right, the problem is `remove_packaged_podman_files` is totally omitted from `setup_environment.sh`.  Don't know don't care how/why...it's likely my fault.  I put it back how intended and CI is happy...